docs/source/logging/logging.md

```diff
@@ -9,10 +9,9 @@ By default, Python only shows logging messages at level `WARNING` and above. Ked
 
 In addition to the `rich` handler defined in Kedro's framework, the [project-side `conf/base/logging.yml`](https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/conf/base/logging.yml) defines three further logging handlers:
```

Comment:
```suggestion
In addition to the `rich` handler defined in Kedro's framework, the [project-side `conf/base/logging.yml`](https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/conf/base/logging.yml) defines two further logging handlers:
```

Please can you just skim through this page to see if there's anything else that should be changed following your change? e.g. maybe we mention the `logs` folder somewhere else, which no longer exists.